README.md

```diff
@@ -29,6 +34,11 @@ In short, most likely you'll not be able to withdraw any money from your busines
 
 Also, you may have unexpected expenses, because life.
 
+As you start receiving the payment, you need to understand that this is you Ltd company money, not your private account.
+You need to set aside money for Corporation Tax, VAT and expense and you can only withdraw from the rest and this will be taxable on both ends - as company profix (so subject to CT) and your profit - subject to 'normal' income taxation.
+
+Finally, as you carry on doing contract work you need to maintain the same level of savings set aside, referred to as a 'warchest'.
+The most tax efficient way of doing this is by keeping the money in the company account. You can then choose what do with the money at a later date - typically this means you can keep paying yourself between contracts.
```

Comment:
This is a great advice, but I think it's a bit out of scope. These things are relevant once you start getting money, this blog mostly focuses on transitioning to contracting. I would be curious to see a separate blog post about managing finances once you get going.

Moreover, if a reader signs up with Crunch, they make it very clear how much money you should put aside for various taxes on the page you see after logging in.